### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by BraisGabin**
Improve our readme in general. Keep the core features on the top of the README.

This implements also adds the suggestion proposed at #4878

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,24 +153,27 @@ As mentioned in...
...
-- [IntelliJ Platform Plugin Template](https://github.com/JetBrains/intellij-platform-plugin-template)
+- [GitHub Action: Setup detekt](https://github.com/marketplace/actions/setup-detekt)
 - [Sonatype Lift](https://github.com/marketplace/muse-dev)
 
 Custom rules and reports from 3rd parties:
 
 - [detekt-verify-implementation](https://github.com/cph-cachet/detekt-verify-implementation) by cph-cachet
 - [detekt-hint](https://github.com/mkohm/detekt-hint) by mkohm is a plugin to detekt that provides detection of design principle violations through integration with Danger
 - [GitLab report format](https://gitlab.com/cromefire_/detekt-gitlab-report)
+- There are mote third-party plugins out there. You can find some of them [in this list](https://github.com/topics/detekt-plugin).
```
Comment (chao2zhang) [inline README.md:176]: 👍

**Review by chao2zhang — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -10,28 +10,21 @@
...
-It operates on the abstract syntax tree provided by the Kotlin compiler.
+Meet _detekt_, a static code analysis tool for the [_Kotlin_ programming language](https://kotlinlang.org/).
+Visit [the project website](https://detekt.dev/) for installation guides, rule descriptions, configuration options and more.
 
 ![detekt in action](website/static/img/tutorial/detekt_in_action.png "detekt in action")
 
 ### Features
 
-- Code smell analysis for your Kotlin projects
-- Complexity reports based on lines of code, cyclomatic complexity and amount of code smells
```
Comment (schalkms) [inline README.md:21]: I wouldn't leave that out, but rather move it a little bit down.

File: README.md (hunk 1 above)
Comment (schalkms) [inline README.md:176]:
```suggestion
- There are more third-party plugins out there. You can find some of them [in this list](https://github.com/topics/detekt-plugin).
```

File: README.md (hunk 2 above)
Comment (BraisGabin) [reply README.md:21]: I rephrased that one to this:

 >  - Support for different report formats: html, markdown, [SARIF](https://sarifweb.azurewebsites.net/) and xml (checkstyle). Is it not enough? You can extend detekt and create your own reports.

But I can keep this one too.

Comment (schalkms) [reply README.md:21]: I think those are two separate things. One is the format (HTML, SARIF...), the other one the reported metric (cyclomatic complexity, code smells...)

Comment (schalkms) [reply README.md:21]: Actually, it should be *number* of code smells. *Amount* is grammatically wrong here.

**Review by cortinico — APPROVED** [review]

**Pull request merged**